manifest.json

```diff
@@ -15,5 +15,13 @@
     }
   ],
 
-  "permissions": ["*://www.quora.com/*"]
+  "permissions": ["*://www.quora.com/*"],
+
+  "background": {
+      "scripts": ["js/background.js"],
+      "persistent": false
+    },
```

Comment:
I removed the third commit from this PR regarding to the background page.

Regarding the background page issue, I think by setting "persistent" to false, it is already an event page? And what `background.js` does is to add a listener for the icon-click event. So I think we don't need to use the alarms API?